File: app/src/main/java/com/cookplan/recipe/import_recipe/search_url/SearchRecipeUrlView.java
package com.cookplan.recipe.import_recipe.search_url;

import com.cookplan.models.network.GoogleRecipe;

import java.util.List;

/**
 * Created by DariaEfimova on 10.06.17.
 */

public interface SearchRecipeUrlView {
    void setResultGoogleSearchList(List<GoogleRecipe> googleRecipes);

    void setError(int errorResource);
}

File: app/src/main/java/com/cookplan/recipe/import_recipe/search_url/SearchRecipeUrlView.kt
package com.cookplan.recipe.import_recipe.search_url

import com.cookplan.models.network.GoogleRecipe

/**
 * Created by DariaEfimova on 10.06.17.
 */

interface SearchRecipeUrlView {
    fun setResultGoogleSearchList(googleRecipes: List<GoogleRecipe>)

    fun setError(errorResource: Int)
}